router/destinationReponseHandler.go

```diff
@@ -147,14 +147,7 @@ func (handler *JSONResponseHandler) IsSuccessStatus(respCode int, respBody strin
 //TXTResponseHandler -- start
 
 //IsSuccessStatus - returns the status code based on the response code and body
-func (handler *TXTResponseHandler) IsSuccessStatus(respCode int, respBody string) (returnCode int) {
```

Comment:
@chandumlg what was the initial intention here?

Reply:
I'd :heart: to know too. It just proxies the code at the moment.

Reply:
Copy paste issue may be. Thanks for the fix.